pool/src/auctions/user_liquidation_auction.rs

```diff
@@ -103,17 +102,13 @@ pub fn create_user_liq_auction_data(
             liquidation_quote.lot.clone(),
             liquidation_quote.bid.clone(),
         );
-        let new_hf = PositionData::calculate_from_positions(e, &mut pool, &user_state.positions)
-            .as_health_factor();
+        let new_data = PositionData::calculate_from_positions(e, &mut pool, &user_state.positions);
 
-        //check if liq is too large
-        if new_hf > 1_1500000 {
-            panic_with_error!(e, PoolError::InvalidLiqTooLarge);
-        }
-        // check if liq is too small
-        if new_hf < 1_0300000 {
-            panic_with_error!(e, PoolError::InvalidLiqTooSmall);
-        }
+        // Post-liq health factor must be under 1.15
+        new_data.require_hf_under(e, 1_1500000, PoolError::InvalidLiqTooLarge);
+
+        // Post-liq heath factor must be over 1.03
+        new_data.require_hf_over(e, 1_0300000, PoolError::InvalidLiqTooSmall);
```

Comment:
we should include a new case or edit the test case for these two errors to have mismatched scalars

Reply:
There are test cases with alternate scalars - both oracle and asset